niri-ipc/src/lib.rs

```diff
@@ -217,6 +217,10 @@ pub enum Action {
     },
     /// Focus the previously focused window.
     FocusWindowPrevious {},
+    /// Focus the next window in most-recently-used order.
+    FocusWindowMruNext {},
+    /// Focus the previous window in most-recently-used order.
+    FocusWindowMruPrevious {},
```

Comment:
As a data point, GNOME calls this "Switch Applications". Then if you manually dig in dconf, the other direction is "switch-applications-backward" with a comment of "Reverse switch applications".